workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Readers/IReader.cs

```diff
@@ -22,6 +21,8 @@ public interface IReader<TSpatialComponentData, TComponentUpdate> : IReader<TSpa
         where TSpatialComponentData : ISpatialComponentData
         where TComponentUpdate : ISpatialComponentUpdate
     {
+        TSpatialComponentData Data { get; }
```

Comment:
Yeah it sounds like we should go back and generate the empty update structs. They won't ever be instantiated, but it will allow us to clean up some things here

Reply:
Wow if there's no real important reason for not generating them that simplifies things a lot... I wonder why we didn't just do that in the first place when we needed them for monobehaviours. Probably best to do that in a refactoring pass over the whole module? (Adding it to the refactor ticket btw)

Reply:
See #193